src/main/webapp/app/plagiarism/manage/exercise-update-plagiarism/exercise-update-plagiarism.component.html

```diff
@@ -82,7 +82,7 @@
             <div class="col">
                 <div class="form-group">
                     <div>
-                        <label class="form-control-label" jhiTranslate="artemisApp.plagiarism.minimumSize"></label>
+                        <label class="form-control-label" jhiTranslate="minimumSizeLabel"></label>
```

Comment:
```suggestion
                        <label class="form-control-label" [jhiTranslate]="minimumSizeLabel"></label>
```
should need this I think?

Reply:
Yes, applied!